apps/src/aiTutor/types.ts

```diff
@@ -64,6 +64,8 @@ export enum Role {
   ASSISTANT = 'assistant',
   USER = 'user',
   SYSTEM = 'system',
+  // only used in Aichat, but our types are currently tangled up :)
+  MODEL_UPDATE = 'update',
```

Comment:
I get this TS error unless I add it:

```
ERROR in ./src/aichat/redux/aichatRedux.ts:159:7
TS2345: Argument of type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aichat/types").ChatCompletionMessage[]' is not assignable to parameter of type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aiTutor/types").ChatCompletionMessage[]'.
  Type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aichat/types").ChatCompletionMessage' is not assignable to type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aiTutor/types").ChatCompletionMessage'.
    Types of property 'role' are incompatible.
      Type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aichat/types").Role' is not assignable to type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aiTutor/types").Role'.
        Property 'MODEL_UPDATE' is missing in type 'import("/Users/benjaminbrooks/code-dot-org/apps/src/aiTutor/types").Role'.
    157 |       newMessageId,
    158 |       message,
  > 159 |       appropriateChatMessages
        |       ^^^^^^^^^^^^^^^^^^^^^^^
    160 |     );
    161 |
    162 |     // Find message in chatMessages and update status.
```

I think because the chatApi is used in aichatRedux, and the type referenced in chatApi is from aiTutor, not aichat. I was kind of hoping that some of the type refactor work @ebeastlake is doing [here](https://github.com/code-dot-org/code-dot-org/pull/57669) might help, but not actually sure it will. In any case, feels like something we can disentangle down the line.